src/preact/index.js

```diff
@@ -43,9 +43,18 @@ export function cloneElement(unusedElement, unusedProps, unusedChildren) {
 /**
  * @param {?PreactDef.VNode} vnode
  * @param {Node} container
+ * @param {?Node=} opt_replaceNode
  */
-export function render(vnode, container) {
-  preact.render(vnode, container, undefined);
+export function render(vnode, container, opt_replaceNode) {
```

Comment:
Is the addition of `opt_replaceNode` an unrelated change?

Reply:
Yeah. Noticed that we missed it at first.